flow/BlobCipher.h

```diff
@@ -67,12 +69,42 @@ class EncryptBuf : public ReferenceCounted<EncryptBuf>, NonCopyable {
 	}
 	uint8_t* begin() { return buffer; }
 
+	StringRef toStringRef() { return StringRef(buffer, logicalSize); }
```

Comment:
How about transforming this to 

`StringRef toStringRef(Arena& arena) { return StringRef(arena, buffer, logicalSize);}`
Better control over StringRef lifecycle management by the caller...

Reply:
The returned StringRef still uses the EncryptBuf's underlying arena, which the caller is supposed to pass when calling `DecryptBlobCipherAes256Ctr::decrypt()`.

Actually I'm thinking we could have `DecryptBlobCipherAes256Ctr::decrypt()` return StringRef directly, since StringRef is more widely used elsewhere. What do you think?

Reply:
Sure, sounds reasonable; StringRef is a wrapper over _uint8_t_ as EncryptBuf, so in principles it is same. 
Alternatively, is for extensibility, we can update EncryptBuf to store StringRef as member instead of uint8_t*, incase, if class needs an update all the caller sites would not be impacted. Thoughts?

Reply:
yup, will do that.

Reply:
will do in later PR.